File: ООП/Homework2/animals_PY/classes/pets/pet.py
from abc import abstractclassmethod
from ..animal import animal

class pet(animal):
    def __init__(self, name, height, weight, coloreye, nickname, breed, vaccination, wool, bday):
        self.nickname = nickname
        self.breed = breed
        self.vaccination = vaccination
        self.wool = wool
        self.bday = bday
        super(pet, self).__init__(name, height, weight, coloreye)

    def pet_me(self):
        print(f"{self.name} {self.nickname} погладь меня ^_^")

    def __str__(self) -> str:
        return (f"{super().__str__()}\nКличка: {self.nickname}\nПорода: {self.breed}\nПрививка: {self.vaccination}\nЦвет: {self.wool}\nДень рождения: {self.bday}")

File: ООП/Homework5/Зоопарк/all_animals/pets/pet.java
package Зоопарк.all_animals.pets;
import Зоопарк.all_animals.main.animals;

public abstract class pet extends animals{
    private String name;
    private String breed;
    private String vaccination;
    private String furColor;
    private String bDay;

    public pet(double height, double weight, String eyeColor, String name, String breed, String vaccinations, String furColor, String bDay) {
        super(height, weight, eyeColor);
        this.name = name;
        this.breed = breed;
        this.vaccination = vaccinations;
        this.furColor = furColor;
        this.bDay = bDay;
    }

    public String getName() {
        return this.name;
    }

    public String getBreed() {
        return this.breed;
    }

    public String getVaccination() {
        return this.vaccination;
    }

    public String getFurColor() {
        return this.furColor;
    }

    public String getBDay() {
        return this.bDay;
    }
    public String Pet() {
        return String.format("=^_^=)");
    }

    @Override
    public String toString() {
        return String.format(
            "%s, Name: %s, Breed: %s, Vaccinations: %s, Fur color: %s, Birth date: %s",
            super.toString(), this.getName(), this.getBreed(), this.getVaccination(), this.getFurColor(), this.getBDay()
        );
    }
}